src/client/gpm_common.c

```diff
@@ -196,7 +196,7 @@ static void gpm_epoll_close(struct gpm_ctx *gpmctx)
 
 static int gpm_epoll_setup(struct gpm_ctx *gpmctx)
 {
-    struct epoll_event ev;
+    struct epoll_event ev = { 0 };
```

Comment:
This should be handled by line 202, no?

Reply:
uhmm right, guess we can drop the memset, as this is more eficient